source/MRViewer/MRRibbonButtonDrawer.cpp

```diff
@@ -11,6 +11,7 @@
 namespace MR
 {
 
+float RibbonButtonDrawer::scaling_ = 1.f;
```

Comment:
why is this?

Reply:
initialize static field (for use in static method)